docs/develop-resources/api-conventions/validations.md

```diff
@@ -0,0 +1,158 @@
+# Direct Resource API Validation
+
+
+# TL;DR
+
+ConfigConnector Direct Resource shall have each `spec` field validated.
```

Comment:
Is Config Connector supposed to be two words?

Reply:
Good call. Changed to `Config Connector` instead (here and else where).